src/Xamarin.Android.Build.Tasks/Utilities/ELFHelper.cs

```diff
@@ -28,7 +28,49 @@ public static bool IsEmptyAOTLibrary (TaskLoggingHelper log, string path)
 				log.LogWarningFromException (ex, showStackTrace: true);
 				return false;
 			}
+		}
+
+		public static bool ReferencesLibrary (string libraryPath, string referencedLibraryName)
+		{
+			if (String.IsNullOrEmpty (libraryPath) || !File.Exists (libraryPath)) {
+				return false;
+			}
+
+			IELF elf = ELFReader.Load (libraryPath);
+
+			// TODO: we need .dynstr
```

Comment:
I'm not sure what this TODO is supposed to be saying when the next statement grabs the "need[ed] .dynstr" section.